Formula/m/mosh.rb

```diff
@@ -63,8 +63,6 @@ def install
 
     # `configure` does not recognise `--disable-debug` in `std_configure_args`.
     system "./configure", "--prefix=#{prefix}", "--enable-completion", "--disable-silent-rules"
-    # Mosh provides remote shell access, so let's run the tests to avoid shipping an insecure build.
-    system "make", "check" if OS.mac? # Fails on Linux.
```

Comment:
this failed earlier

Reply:
also can you propose a better fix for the test?

Reply:
see the check log failures in https://github.com/Homebrew/homebrew-core/actions/runs/15837271549/job/44643440144?pr=227933

Reply:
The test failure was best I can tell a flake. I will have to address this upstream, but I don't have an ETA right now. Disabling the rest is acceptable